testutil/ciao-down/workload.go

```diff
@@ -76,6 +81,40 @@ func (wkld *workload) save(ws *workspace) error {
 }
 
 func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
```

Comment:
There's no problems with having one function that is called just be one other function.  Splitting code up into functions is not just about code re-use.  It's also about readability.  When functions get too long they get hard to read and hard to maintain.  The piece of code you're adding here is fairly self contained and naturally fits in its own function.  Please split.